api/net/Areas/Editor/Controllers/WorkOrderController.cs

```diff
@@ -4,7 +4,7 @@
 using Microsoft.AspNetCore.SignalR;
 using Microsoft.Extensions.Options;
 using Swashbuckle.AspNetCore.Annotations;
-using TNO.API.Areas.Editor.Models.WorkOrder;
+using TNO.API.Areas.Editor.Models.Content;
```

Comment:
This doesn't align with convention.  Normally we create models for a controller so that editing them doesn't impact another controller.